### DISCUSSION TITLE
Huge QField performance difference between different android devices

### DISCUSSION THREAD

**Discussion by nischmi1** [opengisch/QField#2949] [Q&A]
Hi there.
We have a QField project for data collection, including fotos, some calculated fields, some areas of the input forms that are shown/hidden depending on some field values.
Now we noticed a significant performance gap between a Samsung Galaxy Active2 Tablet (Android 9) and an Oppo Find X3 Neo Phone (Android 12): On the Tablet, everything, including just typing some text into an input field, is nerve-wreckingly slow, while on the phone, the usage is fluid and nice. The Tablet is updated to the newest version and I have checked the RAM management.

Does anybody know the reason? Can it be the Android version, or the hardware configuration or do you have some other tipps and tricks and ideas for improving the performance on a mobile device? I don't want to replace it yet, but If I have to, are there device suggestions (tablets)?

**Comment by SandraCardosoEDIA** [comment]
Hi,

We opened the issue Invalid relation message (Issue #2880) describing that these equipments (Samgung Tab Active 2, Android 9) crash in a QField project. We've installed an older QField version (1.9.6) and now it works fine, but we hope this might have a better solution.

**Comment by m-kuhn** [comment]
@nischmi1 how is the performance with the demo projects?
